analysis/study_definition.py

```diff
@@ -21,6 +21,8 @@
     hypertension_codes,  # comorbidities
     chronic_respiratory_disease_codes,
     asthma_codes,
+    systolic_blood_pressure_codes,
```

Comment:
I think it would be better to have your study variables in a separate file and then call them into each of your study definitions as Robin has done [here].(https://github.com/opensafely/bmi-short-data-report/blob/main/analysis/create_variables.py)

Reply:
As discussed, will leave it like this for now but agree it'd be good to use a wildcard (*) at a later stage to tidy study def up.